rotkehlchen/chain/ethereum/modules/monerium/decoder.py

```diff
@@ -21,10 +21,5 @@ def __init__(
             evm_inquirer=ethereum_inquirer,
             base_tools=base_tools,
             msg_aggregator=msg_aggregator,
-            monerium_token_addresses={
-                string_to_evm_address('0x3231Cb76718CDeF2155FC47b5286d82e6eDA273f'),  # EURe
-                string_to_evm_address('0x7ba92741bf2a568abc6f1d3413c58c6e0244f8fd'),  # GBPe
```

Comment:
Hmmm so the last 3 were not checksummed :sweat_smile:  ?

Do we need to take any corrective actions for that?